datadog/dogstatsd/base.py

```diff
@@ -100,6 +100,9 @@ def __init__(
         If set, it is appended to the constant (global) tags of the statsd client.
         :type DD_VERSION: string
 
+        :envvar DD_DOGSTATSD_ENABLE: Enable any statsd metric collection (default True)
+        :type DD_DOGSTATSD_ENABLE: boolean
```

Comment:
If you have accepted the change in README then we need to change it everywhere.
```suggestion
        :envvar DD_DOGSTATSD_DISABLE: Disable any statsd metric collection (default False)
        :type DD_DOGSTATSD_DISABLE: boolean
```